docs/native-code.md

```diff
@@ -0,0 +1,122 @@
+---
+id: native-code
+title: Working with native code on Windows
+---
+
+## What is a React Native for Windows app?
+
+A RNW app is a [Universal Windows Platform app](https://docs.microsoft.com/en-us/windows/uwp/get-started/universal-application-platform-guide) (aka UWP or Windows 10 app).
+
+The Universal Windows Platform allows you to access a set of common functionality on all Windows devices via the [Windows Runtime](https://docs.microsoft.com/en-us/windows/uwp/winrt-components/) (WinRT). WinRT APIs can be accessed from C++ (via [C++/WinRT](https://docs.microsoft.com/en-us/windows/uwp/cpp-and-winrt-apis/)), or via .NET C#. 
+
+#### WinRT support in .NET
+The current publicly supported version of .NET has built-in support for WinRT ([.NET UWP](https://docs.microsoft.com/en-us/windows/uwp/winrt-components/creating-windows-runtime-components-in-csharp-and-visual-basic)) whereas future versions of .NET Core starting with .NET Core 5 will only be able to access WinRT APIs via [C#/WinRT](https://docs.microsoft.com/en-us/windows/uwp/csharp-winrt/) in a way similar to what the C++/WinRT projection enables today. 

```

Comment:
```suggestion
The current publicly supported version of .NET has built-in support for WinRT ([.NET UWP](https://docs.microsoft.com/en-us/windows/uwp/winrt-components/creating-windows-runtime-components-in-csharp-and-visual-basic)). 

Future versions of .NET Core starting with .NET Core 5 will only be able to access WinRT APIs via [C#/WinRT](https://docs.microsoft.com/en-us/windows/uwp/csharp-winrt/) in a way similar to what the C++/WinRT projection enables today.
```

Should we discuss future versions of .NET Core in this doc? I don't understand what the developer action is as a result of this sentence. Is it that their native modules must be written in CsWinRT?

Reply:
.NET Core 5 is about to go live (they already published RC 2). Once they go live, UWP apps written in managed languages should likely be written in .NET Core 5 as .NET UWP will be legacy/deprecated.

Reply:
I still don't understand what the impact is for RNW developers. Will they need to do something special when .NET Core 5 goes live? Or will that be handled by the RNW platform? Will they need to update their native modules?

Reply:
Hmm ok. There still is no official guidance re: .net core 5 and UWP (e.g. https://github.com/microsoft/ProjectReunion/issues/105), it seems like .net core 5 might not support UWP apps "as we know them today" out of the box, so maybe we should have a discussion about what this means separate from this PR.